smart_card_connector_app/src/window-main.js

```diff
@@ -58,7 +58,34 @@ GSC.ConnectorApp.Window.AppsDisplaying.initialize();
 GSC.ConnectorApp.Window.DevicesDisplaying.initialize();
 GSC.ConnectorApp.Window.LogsExporting.initialize();
 
-GSC.PopupWindow.Client.setExplicitWindowTitle();
 GSC.PopupWindow.Client.showWindow();
 
+/** @const */
```

Comment:
I think it makes sense to put this into common/, so that the same functionality could be easily used within other windows.
E.g. GoogleSmartCard.I18n.ProcessAllElementI18nAttributes (that's just an example, maybe you come up with a better name).